doc/build.md

```diff
@@ -138,3 +210,15 @@ PyYAML     3.13
 setuptools 39.0.1 
 shiboken2  5.12.1 
 ```
+
+# How to Load Plug-ins in Maya 
+
+The provided module file (*.mod) facilitates setting various environment variables for plugins and libraries. After the project is successfully built, ```mayaUsd.mod``` is installed inside the install directory. In order for Maya to discover this mod file, ```MAYA_MODULE_PATH``` environment variable needs to be set to point to the location where the mod file is installed.
+Examples:
+```
+set MAYA_MODULE_PATH=C:\workspace\install\RelWithDebInfo
+export MAYA_MODULE_PATH=/usr/local/workspace/install/RelWithDebInfo
+```
+Once MAYA_MODULE_PATH is set, run maya and go to ```Windows->Setting/Preferences-->Plug-in Manager``` to load the plugins.
```

Comment:
Is there a reason the arrows differ `->` and `-->`? If not, maybe pick either one of the two. And maybe add some spaces for readability. E.g.

```Windows -> Setting/Preferences -> Plug-in Manager```